config/routes.rb

```diff
@@ -1,6 +1,10 @@
+require 'resque/server'
+
 Rails.application.routes.draw do
   resources :catalog, param: :druid, only: %i[create update]
   resources :moab_storage, only: %i[index show]
 
+  mount Resque::Server.new, at: "/resque"
```

Comment:
do you want to make this conditional?  like behind a feature flag?  Or does it not matter if we add this all the time.

Reply:
I found one way to do this is with Rails route constraints, like:

`mount Resque::Server.new, at: '/resque', constraints: { host: /worker-machine-name-here/ }`